static/benchmarks/css/landing_page.sass

```diff
@@ -0,0 +1,243 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans')
+$new_green: #45C676
+$new_blue: #47B7DE
+
+
+// navbar
+.navbar
+  height: 75px
+
+.navbar-item.social_icon
+  padding: 0
+  margin-left: 15px
+
+.image.navbar_element
+  max-height: 80%
+  max-width: 80%
+
+
+// main body and components
+.container.main_body
+  background-image: linear-gradient(0deg, #e8f4ea, white)
+
+.button
+  flex: 1 1 auto
+  text-align: center
+  transition: 0.5s
+  color: white
+  border-radius: 16px
+  border-width: 0
+  font-family: 'Open Sans', sans-serif
+  font-size: 15px
+  font-weight: 600
+  padding: 8px 30px
+  margin-right: 20px
+  background-image: linear-gradient(0deg, $new_blue, $new_green)
+
+.big_green
+  color: $new_green
+  font-family: 'Open Sans', sans-serif
+  font-size: 40px
+  font-weight: 800
+
+.text-default
+  color: black
+  font-family: 'Open Sans', sans-serif
+  font-size: 20px
+  font-weight: 200
+  padding-top: 20px
+
+.container.landing::before // controls the landing gradient
+  content: ""
+  position: absolute
+  top: 0
+  left: 0
+  width: 100%
+  height: 100%
+  background-image: linear-gradient(0deg, $new_blue, $new_green)
+  opacity: 0.3
+  z-index: -1
+  border-radius: 40px
+
+.container.landing
+  position: relative  // critical for gradient above
+  margin-top: 100px
+  border-radius: 40px
+
+.image.head_art
+  margin-top: -12px
+  margin-bottom: -12px
+
+.image.logo_green
+  width: 110%
+  max-width: 120%
```

Comment:
make sure this doesn't lead to issues on different devices like phones and tablets